keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -167,15 +167,6 @@ def __init__(
             output_filters=9 * 4, bias_initializer="zeros"
         )
 
-        self.loss_metric = tf.keras.metrics.Mean(name="loss")
-        self.classification_loss_metric = tf.keras.metrics.Mean(
-            name="classification_loss"
-        )
-        self.box_loss_metric = tf.keras.metrics.Mean(name="box_loss")
-        self.regularization_loss_metric = tf.keras.metrics.Mean(
```

Comment:
It looks like the regularization loss and the overall sum loss are not reintroduced anywhere -- am I missing something or does this PR remove those metrics?

Reply:
It's re-introduced in `compile(...weight_decay)`, just like RCNN models